phpunit/functional/DBTest.php

```diff
@@ -370,14 +370,14 @@ public function testTablesHasItemtype()
         $excluded_tables = [
             'glpi_assets_assets', 'glpi_assets_assetmodels', 'glpi_assets_assettypes',
             'glpi_appliancerelations', 'glpi_dropdowns_dropdowns', 'glpi_oauth_access_tokens', 'glpi_oauth_auth_codes',
-            'glpi_oauth_refresh_tokens', 'glpi_stencils',
+            'glpi_oauth_refresh_tokens', 'glpi_stencils', '/^glpi_plugin_formcreator_/'
```

Comment:
This doesn't seem used because you have a hardcoded check for the same name in the block below.